src/Eyes/Db/DbStatus.php

```diff
@@ -6,6 +6,9 @@
 
 class DbStatus extends Status {
 
+  const NO_DRIVER = 'No Mysql driver found, it tried use PDO, mysqli and mysql, but neither are they found.';
```

Comment:
This is the right : "but neither has been found"
https://forum.wordreference.com/threads/neither-of-them-have-has-been-found.2208033/